docs/index.md

```diff
@@ -28,8 +37,25 @@ date when the latest version was retrieved, 💡 means the date was inferred by
     <tr>
         <td>
         {% if entry.prefix %}
+            <script type="application/ld+json">
+            {
+                "@context": "https://schema.org",
+                "@type":"Dataset",
+                "http://purl.org/dc/terms/conformsTo": {
```

Comment:
If you are claiming conformance, then you will need to add some more properties:
- description (this is needed for Google dataset search)
- identifier
- keywords
- license

Reply:
Hmm these metadata aren't currently readily available in the Bioversions repo. In general, it's a huge pain to figure out the license for most of these resources, too (they are not well-annotated). Maybe I can merge this then mark those as todo's for later, which might require bringing in more information from the Bioregistry on the build of the YAML file that the Bioversions website gets autogenerated from

Reply:
I can appreciate the challenges you are referring to. At the moment, Bioschemas does not enforce compliance, but we are planning to move to flagging resources that are not compliant when they claim to be. Perhaps just removing the `conformsTo` lines would be sufficient.

If you can get the descriptions then that will help get your data into the Google dataset search.

I'd

Reply:
Great, will come back to this, thanks so much for the suggestions!